.github/actions/native-models/credential-helper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo username=DavidLawes
```

Comment:
I might want to change this - for now I just created a personal access token in my work github account.